### PR TITLE
Placing back the PROD URLs and updating the test advice to stop DDoS blocking

### PR DESCRIPTION
**Pull request by philgoff**
Placing back the PROD URLs and updating the test advice to stop DDoS blocking

### CODE REVIEW

File: Test.Automation.Engineer.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ Please make this a **single** zip file named `{yourname}-{role-applied-for}.zip`
 
 ### UI Automated Test
 
-Just Eat's consumer-facing test website in the UK is available at [https://uk-publicweb-incapsula-staging-uk.staging-uk.je-labs.com](https://uk-publicweb-incapsula-staging-uk.staging-uk.je-labs.com), which you can use to find takeaway restaurants in a postcode area.
+Just Eat's consumer-facing website in the UK is available at [www.just-eat.co.uk](https://www.just-eat.co.uk/), which you can use to find takeaway restaurants in a postcode area.
+
+_If you are based internationally, you may have to use a more local website to your current location instead of the UK website due to IP restrictions, for example [www.menulog.com.au](https://www.menulog.com.au/) in Australia or [www.just-eat.es](https://www.just-eat.es/) in Spain._
```
Comment (smarkey) [inline Test.Automation.Engineer.md:24]: Do we want to re-introduce this? Or should we deal with that scenario if it arises? Won't it affect the API test too?

File: Test.Automation.Engineer.md [hunk 2]
```diff
@@ -98,21 +107,18 @@ Just Eat has a public API available at <https://smartgateway.staging-uk.je-labs.
...
+- `Check all the restaurants with more than 1 rating should have a star rating greater than 0`
+- `Check all the restaurants with no ratings should have a star rating of 0`
+- `Check 1 restaurant should have a valid URL by checking for a returned HTTP status code of 200 OK`
 
 3. Design 3 more requirements which you think are missing most and implement verification tests.
 
-#### Platform Choice
-
-Feel free to use any programming language, libraries and frameworks you are experienced with.
+**NOTE:** If you call the restaurants endpoint repeatedly within a short period of time you will encounter DDoS protection. Please manage your API calls responsible.
```
Comment (smarkey) [inline Test.Automation.Engineer.md:118]: Change `responsible` to `responsibly`

Comment (smarkey) [reply Test.Automation.Engineer.md:118]: `will encounter DDoS protection and your IP address will be blocked`?

File: Test.Automation.Engineer.md (hunk 1 above)
Comment (philgoff) [reply Test.Automation.Engineer.md:24]: @smarkey Actually, you are right.  This is not a true statement. That will teach me for copy and pasting what was there before.

**Review by smarkey — APPROVED** [review]

**Review by eignatyev — APPROVED** [review]

**Review by martincostello — APPROVED** [review]

**Pull request merged**